### DISCUSSION TITLE
obx8 v2.0

### DISCUSSION THREAD

**Discussion by alainTHIRY** [christofmuc/KnobKraft-orm#335] [Q&A]
i try this soft, it works with sequential p10 but not with obx8, maybe because the os version of obx8 (it has been updated to v2.0 in july).

**Comment by christofmuc** [comment]
Let us try to find the information on OB-X8. As far as I remember, the manual did not contain the sysex info, unlike all the Sequential synths. Maybe we can ask the Support for that info, in the past they have been very good and helpful.

What we can do to start is that you record a preset using the procedure mentioned in the manual and attach it here. 

https://oberheim.com/wp-content/uploads/2022/08/OB-X8-Users-Guide-1.0.pdf

When I understand this correctly, the OB-X8 is able to load and send single programs. What we don't know yet is how the computer could ask for a preset to be sent by the synth.

**Reply by christofmuc** [reply]
Oh, found it:

https://forum.sequential.com/index.php/topic,6483.0.html

The new user guide has a section on all the usual sysex implementation details at it is using the same implementation and even the Sequential MIDI id. So updating the KnobKraft should be really simple, I'll have a look!

**Reply by christofmuc** [reply]
The manual was copied and pasted from the Pro-3 manual, and I don't trust the information. I have started a topic in the Sequential forum on this, let's see if we can learn more: https://forum.sequential.com/index.php/topic,8073.0.html

**Comment by alainTHIRY** [comment]
thank you for your cooperation! you're right the ID in sysex is not correct, strange...

**Comment by alainTHIRY** [comment]
[prest.zip](https://github.com/user-attachments/files/16648886/prest.zip)
here a preset dump from obx8

**Comment by christofmuc** [comment]
Hey, sorry I forgot about this as this happened two days before my vacation. I am back now and looked at the preset you provided, and it actually loads ok, it can extract the name "Jersey Girl B", and recognizes this as a program dump for the OB-X8. 

Maybe the auto detection doesn't work? Can you check that it confirms the synth his detected properly? You can append the MIDI log here after the autodetection run, so we can see if the synth replies as expected. When you have both the Prophet and the OBX-8 connected and setup in KnobKraft, after auto detect both shoul show a green bar below their button in the top synth selector bar?

**Comment by alainTHIRY** [comment]
Hi, it is on my side to not be at home, i look at that in middle of next week… :)AlainLe 4 oct. 2024 à 22:09, Christof ***@***.***> a écrit :﻿
Hey, sorry I forgot about this as this happened two days before my vacation. I am back now and looked at the preset you provided, and it actually loads ok, it can extract the name "Jersey Girl B", and recognizes this as a program dump for the OB-X8.
Maybe the auto detection doesn't work? Can you check that it confirms the synth his detected properly? You can append the MIDI log here after the autodetection run, so we can see if the synth replies as expected. When you have both the Prophet and the OBX-8 connected and setup in KnobKraft, after auto detect both shoul show a green bar below their button in the top synth selector bar?

**Reply by christofmuc** [reply]
@alainTHIRY Did you actually had another look at this? I'd like to confirm the OB-X8 adaptation works. Cheers!

**Comment by discombob93** [comment]
He's got basic functionality now working for OB-X8!  Exciting... now we just need drag and drop everywhere with multi-selection via shift or ctrl... like PolyBrute Connect, whose excellent interface ideas in version 3 and higher should be applied to general librarians for other synths.

https://forum.sequential.com/index.php/topic,8073.0.html


NOTE: "Jersey Girl B" is supposed to read "Jersey Girl Bass" as the full preset name.  The longer Preset names are being truncated in the interface but still seem to be pushed to the synth OK.

**Reply by christofmuc** [reply]
Ah, the name length can be adjusted easily. How many characters are possible?

**Reply by discombob93** [reply]
I just counted out a possibility of 20 characters on my OB-X8 OLED display.

**Comment by alainTHIRY** [comment]
Hi, sorry for no answer sooner,   I will look at this when I will be at homeAlainLe 19 janv. 2025 à 22:20, Christof ***@***.***> a écrit :﻿
@alainTHIRY Did you actually had another look at this? I'd like to confirm the OB-X8 adaptation works. Cheers!

**Comment by christofmuc** [comment]
This was now released as part of release 2.5.2 today.
